assets/templates/partials/census/get-data.tmpl

```diff
@@ -0,0 +1,34 @@
+{{ $length := len .Version.Downloads }}
+<section id="get-data" class="u-mt-l u-mb-l" aria-label="{{ localise "GetData" .Language 1 }}">
+    <h2 class="u-fw-b u-mt-l u-pb-no u-pt-no">{{ localise "GetData" .Language 1 }}</h2>
+    <p class="u-fs-r u-mt-s u-mb-s u-pt-no u-pb-s">{{ localise "GetDataLeadText" .Language 1 }}</p>
+    <ul class="list list--bare list--icons u-fs-r u-pb-s">
+        {{ range $i, $el := .Version.Downloads }}
+            <li class="list__item u-mt-s u-mb-no u-pl-no u-pb-s u-pt-no u-fs-r{{if notLastItem $length $i }} u-bb{{end}}">
+                <span class="list__prefix">
+                    <svg
```

Comment:
Just a suggestion as we're only using this icon here atm, but consider extracting this SVG into its own partial so it can be used across templates

Reply:
I was thinking that all the icons could be placed in a more generic location

`assets/templates/icons` and the sharing icons in their own subfolder `assets/templates/icons/share`